### PR TITLE
Fix verify page loader

### PR DESCRIPTION
**Pull request by selankon**

### CODE REVIEW

File: components/pages/verify/single-page.tsx [hunk 1]
```diff
@@ -33,14 +30,18 @@ const VerifySinglePage = ({ voteNullifier } : { voteNullifier: string }) => {
...
+  const envelopeFound = envelope !== null && envelope !== undefined
+  const isLoading = (loading && !envelope) || (urlNullifier && envelope === undefined)
 
   return (
     <>
-    <VerifyPageContainer>
+      <VerifyPageContainer>
         <VerifyPage
-          minified={envelope !== null && envelope !== undefined}
+          minified={envelopeFound || envelopeNotFound}
```
Comment (elboletaire) [inline components/pages/verify/single-page.tsx:44]: Wouldn't this be the same as setting `minified={true}`? Or do we have Schrödinger envelopes?

File: components/pages/verify/single-page.tsx [hunk 2]
```diff
@@ -56,25 +57,18 @@ const VerifySinglePage = ({ voteNullifier } : { voteNullifier: string }) => {
...
-                <Then>
-                <FlexContainer
-                  alignItem={FlexAlignItem.Center}
-                  justify={FlexJustifyContent.Center}
-                >
-                  <h2>{i18n.t('envelopes.details.envelope_not_found')}</h2>
-                </FlexContainer>
-                </Then>
+              <If condition={envelopeNotFound}>
+                <h2>{i18n.t('envelopes.details.envelope_not_found')}</h2>
```
Comment (elboletaire) [inline components/pages/verify/single-page.tsx:71]: Careful, the `If` component should always have a `Then` or `Else` children. Use `When` if you want to use less components in a case like this (or `Unless` in the opposite cases).

**Review by elboletaire — CHANGES_REQUESTED** [review]

File: components/pages/verify/single-page.tsx (hunk 1 above)
Comment (selankon) [reply components/pages/verify/single-page.tsx]: Not exactly a Schrödinger envelope, on the initial state `envelopeFound` and `envelopeNotFound` are false. Refactoring the variable name should be more intuitive

```diff
-         minified={envelopeFound || envelopeNotFound}
+        minified={envelopeLoaded || envelopeNotFound}
```

bfa23dc3c6ab43177fb167db6380df2a781b381d

File: components/pages/verify/single-page.tsx (hunk 2 above)
Comment (selankon) [reply components/pages/verify/single-page.tsx:71]: Fix 46f39297137e863f4bbfc2e6e470e6e48f48e034

**Review by elboletaire — APPROVED** [review]

**Pull request merged**